official/vision/beta/projects/yolo/modeling/decoders/yolo_decoder_test.py

```diff
@@ -0,0 +1,148 @@
+# Lint as: python3
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
It's 2021 but I'm assuming this is boilerplate

Reply:
Replaced by 2021 comments